django_project/certification/templates/certifying_organisation/detail.html

```diff
@@ -268,7 +268,7 @@ <h5>No courses are defined, but you can <a
             {% endif %}
         </td>
 
-        {% if user_can_delete %}
+        {% if user_can_delete and user_can_create %}
```

Comment:
I'm failing to understand this one. It's for a "delete" button, why do you need the create too?

Reply:
We have 2 actions in that div.
For delete as well as for creating.

Reply:
You need to move the if for each individual button maybe?

Reply:
>You need to move the if for each individual button maybe?

We could but it works alright still like this as well as the early permission checks were implemented like that as well. 

It works alright for the delete check only,
I have cleaned it up.

Reply:
I think it makes sense to as he who can delete means can also update or create.